src/main/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBase.scala

```diff
@@ -158,21 +161,44 @@ trait DataFrameSuiteBaseLike extends SparkContextProvider
   }
 
   /**
-   * Zip RDD's with precise indexes. This is used so we can join two DataFrame's
-   * Rows together regardless of if the source is different but still compare
-   * based on the order.
-   */
+    * Zip RDD's with precise indexes. This is used so we can join two DataFrame's
+    * Rows together regardless of if the source is different but still compare
+    * based on the order.
+    */
   private[testing] def zipWithIndex[U](rdd: RDD[U]) = {
-    rdd.zipWithIndex().map{ case (row, idx) => (idx, row) }
+    rdd.zipWithIndex().map { case (row, idx) => (idx, row) }
   }
 
   def approxEquals(r1: Row, r2: Row, tol: Double): Boolean = {
     DataFrameSuiteBase.approxEquals(r1, r2, tol)
   }
+
+  def schemaEquals(expected: DataFrame, result: DataFrame): Unit = {
```

Comment:
I agree `assertSchemaEquals` does make more sense. Will rename to conform with standard.